bundles/org.openhab.binding.deutschebahn/src/main/resources/OH-INF/i18n/deutschebahn_de.properties

```diff
@@ -18,12 +18,14 @@ thing-type.deutschebahn.train.group.trip.description = Enthält alle Information
 
 # thing types config
 
-thing-type.config.deutschebahn.timetable.accessToken.label = Zugriffsschlüssel
-thing-type.config.deutschebahn.timetable.accessToken.description = Zugriffsschlüssel für die Timetable V1 API aus dem Developer-Portal der DeutschenBahn.
+thing-type.config.deutschebahn.timetable.clientId.label = Client Id
```

Comment:
Translations should be provided through Crowdin. Please see https://github.com/openhab/openhab-addons/blob/main/README.md#translations.

Reply:
Thanks for your review.
I've reverted the changes and applied them in crowdin.